hooks/capacitorCopyPreferences.js

```diff
@@ -0,0 +1,245 @@
+        const application = root.find("application");
+        let metaData = application.findall("meta-data");
+        const existing = metaData.find(el => el.get("android:name") === "com.google.android.gms.wallet.api.enabled");
+        if (existing) {
+            existing.set("android:value", "true");
+        } else {
+            const newMeta = new et.Element("meta-data");
+            newMeta.set("android:name", "com.google.android.gms.wallet.api.enabled");
+            newMeta.set("android:value", "true");
+            application.append(newMeta);
+        }
+        
+        fs.writeFileSync(manifestPath, etreeManifest.write({ xml_declaration: true, indent: 4 }));
+    }
+}
+
+function configureIOS(paymentConfig) {
+    const plist = require("plist");
+    const xcode = require('xcode');
+
+    let merchant_id = "";
+    let merchant_name = "";
+    let merchant_country_code = "";
+    let payment_allowed_networks = [];
+    let payment_supported_capabilities = [];
+    let payment_supported_card_countries = [];
+    let shipping_supported_contacts = [];
+    let billing_supported_contacts = [];
+    let payment_gateway = "";
+    let payment_request_url = "";
+    let stripe_publishable_key = "";
+
+    paymentConfig.app_configurations.forEach((configItem) => {
+        if (configItem.service_id == ServiceEnum.ApplePay) {
+            let error_list = [];
+
+            merchant_id = validateAndAssignField(configItem, "merchant_id", error_list, "Merchant Id");
+            merchant_name = validateAndAssignField(configItem, "merchant_name", error_list, "Merchant Name");
+            merchant_country_code = validateAndAssignField(configItem, "merchant_country_code", error_list, "Merchant Country");
+            payment_allowed_networks = validateAndAssignField(configItem, "payment_allowed_networks", error_list, "Payment Allowed Networks");
+            payment_supported_capabilities = validateAndAssignField(configItem, "payment_supported_capabilities", error_list, "Payment Supported Capabilities");
+            shipping_supported_contacts = configItem.shipping_supported_contacts;
+            billing_supported_contacts = configItem.billing_supported_contacts;
+            payment_supported_card_countries = configItem.payment_supported_card_countries;
+
+            if (configItem.tokenization) {
+                payment_gateway = validateAndAssignField(configItem.tokenization, "gateway", error_list, "Payment Gateway Name");
+                payment_request_url = validateAndAssignField(configItem.tokenization, "requestURL", error_list, "Payment Request URL");
+                stripe_publishable_key = validateAndAssignField(configItem.tokenization, "stripePublishableKey", error_list, "Stripe's Publishable Key");
+
+                if (!stripe_publishable_key && payment_gateway && payment_gateway.toLowerCase() === "stripe") {
+                    error_list.push("Stripe's Publishable Key");
+                }
+            }
```

Comment:
Doesn't this add the "Stripe's Publishable Key" error twice? Or more importantly, add a "Stripe's Publishable Key" even if strip is not being used?

I'm thinking if instead this should be:

```suggestion
                stripe_publishable_key = configItem.tokenization.stripePublishableKey;

                if (!stripe_publishable_key && payment_gateway && payment_gateway.toLowerCase() === "stripe") {
                    error_list.push("Stripe's Publishable Key");
                }
            }
```

Or something that resembles more what the existing hook has:

```javascript
                if (configItem.tokenization.stripePublishableKey != null && configItem.tokenization.stripePublishableKey !== "") {
                    stripe_publishable_key = configItem.tokenization.stripePublishableKey;
                } else if (payment_gateway.toLowerCase() === "stripe") {
                    error_list.push('Stripe\'s Publishable Key');
                }
```